Source/WebCore/loader/archive/cf/LegacyWebArchive.cpp

```diff
@@ -506,7 +506,7 @@ RetainPtr<CFDataRef> LegacyWebArchive::createPropertyListRepresentation(const Re
 
 #endif
 
-RefPtr<LegacyWebArchive> LegacyWebArchive::create(Node& node, NOESCAPE const Function<bool(LocalFrame&)>& frameFilter, const Vector<MarkupExclusionRule>& customMarkupExclusionRules, const String& mainResourceFilePath, bool saveScriptsFromMemoryCache, ShouldArchiveSubframes shouldArchiveSubframes)
+RefPtr<LegacyWebArchive> LegacyWebArchive::create(Node& node, NOESCAPE const Function<bool(LocalFrame&)>& frameFilter, const LegacyWebArchive::ArchiveOptions& options)
```

Comment:
Surely we don't need ` LegacyWebArchive::` here and can just use `const ArchiveOptions&` directly?

Reply:
Also, I'm wondering if we could take a `ArchiveOptions&&` here so the implementation can just modify the parameter instead of making a copy. Is there really a call site that cannot move the options in?